KsApi/models/graphql/GraphCommentsEnvelopeTests.swift

```diff
@@ -67,6 +68,7 @@ final class GraphCommentsEnvelopeTests: XCTestCase {
 
     do {
       let envelope = try JSONDecoder().decode(GraphCommentsEnvelope.self, from: data)
+      XCTAssertEqual(envelope.slug, "jadelabo-j1-beautiful-powerful-and-smart-idex-3d-printer")
```

Comment:
Personal style/format thing here but I would put this toward the bottom of the `do` just for organization reasons.